### PR TITLE
custom servey page 1 and page 2 model structure implementation

### PR DESCRIPTION
**Pull request by vivektwr**

### CODE REVIEW

File: src/main/java/com/synerzip/survey_builder/models/CustomServey.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "custom_servey")
+public class CustomServey {
+
+	@Id
+	private ObjectId customServeyId;
```
Comment (haripcce) [inline src/main/java/com/synerzip/survey_builder/models/CustomServey.java]: 1. rename from Custom Survey to Survey.
2. customServeyId to id
3. similary rename other fields containing prefix as surevey
4. re design  the survey entity as per the schema design a) e.g questions should be in seperate schema

File: src/main/java/com/synerzip/survey_builder/models/ServeyStatus.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.synerzip.survey_builder.models;
+
+public enum ServeyStatus {
```
Comment (haripcce) [inline src/main/java/com/synerzip/survey_builder/models/ServeyStatus.java]: spelling mistake ServeyStatus

File: src/main/java/com/synerzip/survey_builder/models/ServeyType.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package com.synerzip.survey_builder.models;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "servey_type")
```
Comment (haripcce) [inline src/main/java/com/synerzip/survey_builder/models/ServeyType.java]: spelling mistake for survey

File: src/main/java/com/synerzip/survey_builder/models/ServeyType.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+public class ServeyType {
+
+	@Id
+	private ObjectId typeId;
```
Comment (haripcce) [inline src/main/java/com/synerzip/survey_builder/models/ServeyType.java]: No need for separate identifier for id every time. Rename to id everywhere

File: pom.xml [hunk 5]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (haripcce) [inline pom.xml:4]: Add proper description details for the project.

add migration for mongo db and mysql

**Review by haripcce — CHANGES_REQUESTED** [review]

**Comment by vivektwr** [comment]
pull request merged

**Pull request merged**